spec/preprocessinator_includes_handler_spec.rb

```diff
@@ -61,116 +61,126 @@
+  #                        'some_other_lib/some_header2.h',
+  #                        'Build/temp/_test_DUMMY.c']
+  #   end
 
-    it 'should correctly filter secondary dependencies' do
-      # create test state/variables
-      # mocks/stubs/expected calls
-      expect(@configurator).to receive(:extension_header).and_return('.h')
-      expect(@configurator).to receive(:extension_source).and_return('.c')
-      # execute method
-      results = subject.extract_shallow_includes(%q{
-        _test_DUMMY.o: Build/temp/_test_DUMMY.c \
-          source\some_header1.h \
-          source\some_lib\some_header2.h \
-          source\some_lib1\some_lib\some_header2.h \
-          source\some_other_lib\some_header2.h \
-          source\some_other_lib\another.h \
-          @@@@some_header1.h \
-          @@@@some_lib/some_header2.h \
-          @@@@lib/some_header2.h \
-          @@@@some_other_lib/some_header2.h
-      })
-      # validate results
-      expect(results).to eq ['some_header1.h',
-                         'some_lib/some_header2.h',
-                         'some_other_lib/some_header2.h',
-                         'Build/temp/_test_DUMMY.c']
-    end
+  #   it 'exclude annotated headers with no matching "real" header' do
+  #     # create test state/variables
+  #     # mocks/stubs/expected calls
+  #     expect(@configurator).to receive(:extension_header).and_return('.h')
+  #     expect(@configurator).to receive(:extension_source).and_return('.c')
+  #     # execute method
+  #     results = subject.extract_includes(%q{
+  #       _test_DUMMY.o: Build/temp/_test_DUMMY.c \
+  #         source/some_header1.h \
+  #         @@@@some_header1.h \
+  #         @@@@some_lib/some_header2.h
+  #     })
+  #     # validate results
+  #     expect(results).to eq ['some_header1.h', 'Build/temp/_test_DUMMY.c']
+  #   end
+
+  #   it 'should correctly filter secondary dependencies' do
+  #     # create test state/variables
+  #     # mocks/stubs/expected calls
+  #     expect(@configurator).to receive(:extension_header).and_return('.h')
+  #     expect(@configurator).to receive(:extension_source).and_return('.c')
+  #     # execute method
+  #     results = subject.extract_includes(%q{
+  #       _test_DUMMY.o: Build/temp/_test_DUMMY.c \
+  #         source\some_header1.h \
+  #         source\some_lib\some_header2.h \
+  #         source\some_lib1\some_lib\some_header2.h \
+  #         source\some_other_lib\some_header2.h \
+  #         source\some_other_lib\another.h \
+  #         @@@@some_header1.h \
+  #         @@@@some_lib/some_header2.h \
+  #         @@@@lib/some_header2.h \
+  #         @@@@some_other_lib/some_header2.h
+  #     })
+  #     # validate results
+  #     expect(results).to eq ['some_header1.h',
+  #                        'some_lib/some_header2.h',
+  #                        'some_other_lib/some_header2.h',
+  #                        'Build/temp/_test_DUMMY.c']
+  #   end
```

Comment:
Candidate to remove them, since they're commented out :yum:

Reply:
:) - done